### ISSUE TITLE
Valid signature is marked as invalid

### ISSUE THREAD

**Issue by PRabahy** [mailvelope/mailvelope#317]
On https://tahoe-lafs.org/pipermail/tahoe-dev/2015-May/009431.html, I get

Wrong signature of Paul Rabahy <PRabahy@gmail.com> (Key ID: D1CBA2A21BCD88F6)

, but when I look at the same message in Thunderbird with Enigmail, I get

Good signature from Paul Rabahy <PRabahy@gmail.com>
Key ID: 0x1BCD88F6 / Signed on: 5/27/2015 5:43 PM
Key fingerprint: EA69 5E0C E2D0 DCB0 D651 67A8 D1CB A2A2 1BCD 88F6

I also looked at this same message through the gmail web interface and got the same result (mailvelope says wrong signature).

I tried a private test going the other way (signed by mailvelope) and both Enigmail and mailvelope see it a good signature.

**Comment by soderluk** [comment]
This is still an issue in June 2016.

**Comment by Izzette** [comment]
I am still getting this issue in November 2016.  This is critical functionality, where's the attention, where's the fix?

**Comment by bjmgeek** [comment]
This is still an issue.  I am using v1.7.1 on Firefox ESR 45.8.0 on Debian Jessie, on x86_64.

Here's an example of a message that fails to verify in gmail with mailvelope (even when I click "view original" in gmail:

````
-----BEGIN PGP SIGNED MESSAGE-----
Hash: SHA256

I realized I had an extra libgpgme.so.11 sitting around in
/usr/local/lib. Once I removed it, mutt worked fine. This bug report
may be closed.
-----BEGIN PGP SIGNATURE-----

iHUEARYIAB0WIQTu0BWAE9wubW4AHqQ3uVB6z/IBbgUCWNkIFgAKCRA3uVB6z/IB
bpXOAP9TSOF6k0QZ7NYEBavzA3JvjG2/AwzoLHY7O7wiYqroPwEAxMVZU9L1kT6a
1Fr+Aup2ZfZhytegW3XzZINKhdPCMA+IdQQBEQgAHRYhBPnEu3YOeD8N7BCmimuO
s6Blz7qpBQJY2QgyAAoJEGuOs6Blz7qpLu8A/jz8g4XoIcMA1rGLdMBGsM5+dM9Y
oOgyASaJb/jy/aqAAP0bkqPz0B3RcqscF2moVxDhamO9lwA31wJ1qvmtW1p5MA==
=tfxg
-----END PGP SIGNATURE-----
````

**Comment by sbaker-gsc** [comment]
Same problem with gmail in Chrome 61 + Mailvelope v1.8.0 - but... If you "view original message" then and only then it validates. It's a pain to do that every time, but at least it shows that the message is valid for those cases where it's essential.

Just FYI in case it helps anyone looking into this.

**Comment by xnox** [comment]
Nope, fails to validate for me =( with/without gnupg, with/without pgp/mime, fails to even validate emails it sent.

**Comment by easz** [comment]
Mailvelope 4.40
Firefox 87.0
macOS 10.14
mailbox.org

Mailvelope option enable: Sign all outgoing messages.

it always shows "Invalid signature"

**Comment by HappyFool123** [comment]
Still a issue in 2022. Great work!

**Comment by ivandimitrov8080** [comment]
and 2023...
